packages/filebrowser-extension/src/index.ts

```diff
@@ -232,6 +230,23 @@ function activateBrowser(app: JupyterLab, factory: IFileBrowserFactory, restorer
  * Add the main file browser commands to the application's command registry.
  */
 function addCommands(app: JupyterLab, tracker: InstanceTracker<FileBrowser>, browser: FileBrowser): void {
+  const getBrowserForPath = (path: string): FileBrowser => {
+    const driveName = app.serviceManager.contents.driveName(path);
+
+    if (driveName) {
+      let browserForPath = tracker.find((fb: FileBrowser) => fb.id.toLowerCase().startsWith(driveName.toLowerCase()));
+
+      if (!browserForPath) {
+        // warn that no filebrowser could be found for this driveName
+        console.warn(`${CommandIDs.navigate} failed to find filebrowser for path: ${path}`); return;
```

Comment:
Will do. For various reasons I was inspired to a little stylistic flourish, but I have no problem with splitting it up either.

Originally I had it as a single line `return console.warn(...);`, but that isn't valid code. I guess I had assumed that even if `console.warn()` didn't explicitly define a return type, it would still return `null` (most of the languages I'm familiar with do something similar in one form or another). Just for the sake of my own curiosity, do functions in JS/TS not have this kind of default return value?